docs/src/faq.md

```diff
@@ -1,5 +1,18 @@
 # Frequently asked questions
 
+```@meta
+DocTestSeup =
+    using Pkg
+    Pkg.add("Mimi")
+    Pkg.add("DelimitedFiles")
+    Pkg.add("Distributions")
```

Comment:
We shouldn't install the packages in here, instead they should be added to [this file](https://github.com/mimiframework/Mimi.jl/blob/master/docs/Project.toml), then they will be installed automatically for the doc build.

Reply:
Ah ok good I was going to ask about this.  I need to put a little thought into how to incorporate these tests, with one-off examples they make sense but in piecewise tutorials with text in between code blocks it’s a little less obvious how to do this efficiently.

Reply:
@davidanthoff should the `[compat]` line of the Project.toml file have a line for everything in `[deps]`?

Reply:
as long as it's not in base